Sources/iYuga/Yuga.swift

```diff
@@ -351,8 +351,10 @@ public class Yuga {
         }
         let pattern = "([0-9]{2})([0-9]{2})?([0-9]{2})?"
         let m = str.groups(for: pattern)
-        let value = m[0] + ((m.count > 1) ? (":" + m[2]) : ":00")
-        valMap[pre + "time"] = value
+        if (m.count > 0) {
```

Comment:
Is it the same in java or is the regex outputs different in swift & java ?

Reply:
Its the same but if m is empty then the application will crash. So I have added a check here. We already do this check in Java.